### PR TITLE
[MIG] stock_valued_picking_report: Migration to 10.0

### PR DESCRIPTION
**Pull request by chienandalu**
Stock Valued Picking Report
===========================

Add amount information to Delivery Slip report.
You can select at partner level if picking list report must be valued or not.

cc @Tecnativa

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
@rafaelbn @cubells please review

**Review by cubells — APPROVED** [review]
:+1: 

LGTM

Code review, tests and translations.

Good job @chienandalu

File: stock_valued_picking_report/README.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+Stock Valued Picking Report
+===========================
+
+Add amount information to Delivery Slip report.
+You can select at partner level if picking list report must be valued or not.
```
Comment (rafaelbn) [inline stock_valued_picking_report/README.rst:10]: If the partner by default has set not to receive valued picking report, is it possible to choose this option in the stock picking?

File: stock_valued_picking_report/README.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+Configuration
+=============
+
+#. Go to *Customers > (select one of your choice) > Sales & Purchases*.
+#. Set *Valued picking* field on.
```
Comment (rafaelbn) [inline stock_valued_picking_report/README.rst:16]: I'm testing in runbot but @chienandalu I cannot find this option even with developer mode activated and all permissions, check:

**Review by rafaelbn — COMMENTED** [review]
I have 2 comments please @chienandalu

File: stock_valued_picking_report/README.rst (hunk 2 above)
Comment (rafaelbn) [reply stock_valued_picking_report/README.rst:16]: OK, this is because ASUSTek is not a customer

**Review by rafaelbn — APPROVED** [review]
Tested in runbot 👍

File: stock_valued_picking_report/README.rst (hunk 1 above)
Comment (chienandalu) [reply stock_valued_picking_report/README.rst:10]: @rafaelbn No, it's not possible. It checks it at document level. Maybe it wouldn't be hard to have an option in the order to override partner's preference.

**Pull request merged**